xalienfs.sh

```diff
@@ -3,79 +3,73 @@ version: "%(tag_basename)s"
 tag: v1.0.14r1-alice3
 source: https://github.com/alisw/xalienfs.git
 requires:
- - XRootD
- - "OpenSSL:(?!osx)"
- - "osx-system-openssl:(osx.*)"
- - AliEn-Runtime
+  - zlib
+  - libxml2
+  - "OpenSSL:(?!osx)"
+  - "osx-system-openssl:(osx.*)"
+  - AliEn-CAs
+  - ApMon-CPP
+  - UUID
+  - XRootD
```

Comment:
As far as I can tell, xalienfs directly depends on XRootD and OpenSSL. Are the rest really required? If you're not sure, could you remove them and see if that breaks the build?

Reply:
sure, i left UUID as there is a usage of the form of `${UUID_ROOT:+-I$UUID_ROOT/include -L$UUID_ROOT/lib}` in CXXFLAGS